package.json

```diff
@@ -114,6 +114,7 @@
     "@folio/eslint-config-stripes": "^6.1.0",
     "@folio/stripes": "^7.0.0",
     "@folio/stripes-cli": "^2.4.0",
+    "@folio/stripes-testing": "^4.3.1000004901",
```

Comment:
It is strange that the `4.3.1` version is installed, although the [latest version](https://github.com/folio-org/stripes-testing/releases) is `4.2.0`.